media/json-jettison/src/main/java/org/glassfish/jersey/jettison/JettisonConfig.java

```diff
@@ -97,13 +97,15 @@ private Builder(final Notation notation) {
          * @return a non-null {@link JettisonConfig} instance
          */
         public JettisonConfig build() {
```

Comment:
What about making build an abstract method, remove strange `instanceof` a implement it in sub-classes?